examples/basic/fractal.js

```diff
@@ -8,7 +8,7 @@ const paths = {
 };
 
 // Create a new theme instance with custom config options
-const theme = eclTheme({ skin: 'blue' });
```

Comment:
There are some occurrences of `// Custom (to be deleted asap)`
It might be the right time to get rit of all of these.